.woodpecker/main.yml

```
@@ -50,84 +50,86 @@ pipeline:
       - (cd web/; go generate ./...)
       - make release
 
-  # publish-server-alpine:
-  #   image: plugins/docker
-  #   repo: woodpeckerci/woodpecker-server
-  #   dockerfile: Dockerfile.alpine
-  #   secrets: [ docker_username, docker_password ]
-  #   tag: [ alpine ]
-  #   when:
-  #     branch: master
-  #     event: push
-
   build-cli:
     image: golang:1.16
     commands: make release-cli
 
   publish-server:
     image: plugins/docker
     repo: woodpeckerci/woodpecker-server
+    dockerfile: docker/Dockerfile.server
     secrets: [docker_username, docker_password]
     tag: [latest]
     when:
       branch: master
       event: push
 
+  publish-server-alpine:
+    image: plugins/docker
+    repo: woodpeckerci/woodpecker-server
+    dockerfile: docker/Dockerfile.server.alpine
+    secrets: [ docker_username, docker_password ]
+    tag: [latest-alpine]
```

I would change it to `next-alpine` as latest is normally the latest stable release.
```suggestion
    tag: [next-alpine]
```